docs/src/monitoring.md

```diff
@@ -90,6 +91,49 @@ spec:
     and will be removed in the future. Please use the label `cnpg.io/cluster`
     instead to select the instances.
 
+### Enabling TLS on the Metrics Port
+
+To enable TLS communication on the metrics port, configure the `spec.monitoring.tls.enable`
+setting to `true`. This setup ensures that the metrics exporter uses the same
+server certificates as PostgreSQL to secure communication on port 5432.
+
+!!! important
+    Changing the `spec.monitoring.tls.enable` value will cause all instances to restart.
+    Plan accordingly to minimize downtime.
```

Comment:
"All instances to restart" seems very chaotic, where this is our well known operation of a rolling restart.
And "minimize downtime" ... with a well performed rolling restart, there would be none, right?

Reply:
What about?

```suggestion
    Changing the `spec.monitoring.tls.enable` value will cause a rollout of all instances in the cluster.
```